data/human/hails.txt

```diff
@@ -1568,6 +1570,109 @@ phrase "friendly civilian"
 		"Apparently people think we'll work for exposure."
 		"Not to mention this ship is not designed for a job like it."
 
+phrase "friendly civilian"
+	word
+		"Don't go for"
+		"Don't participate in"
+		"Stay away from"
+		"Watch out for"
+	word
+		" "
+	word
+		"badly organized races"
+		"inheritance scams"
+		"shady transport jobs"
+		"hitman transports"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"I lost 20000 credits on one of those."
+		"I nearly got killed in one."
+		"The Navy almost arrested me."
+		"Those can ruin your life!"
+		"They are usually too good to be true."
+		"They take your money and you won't get anything in return."
+
+phrase "friendly civilian"
+	word
+		"I have a delivery for"
+		"I'm looking for"
+		"My passengers need to go to"
+	word
+		" "
+	word
+		"Dune"
+		"Earth"
```

Comment:
Earth?

Reply:
Yes, what is wrong with somebody looking for Earth? Some pilot born on Trinket might never have visited Earth before and might just not know how to get there.

The "have you ever heard of a planet with that name?" could be a bit odd in combination with Earth, because you would expect most people to at least have heard of Earth through the news. But I can imagine that some people didn't or that the person just wants to brag about going to Earth using this text.